<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Message:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>21</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Socket</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pipe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>共享存储区</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>通信方式：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
